### DISCUSSION TITLE
Why migrate from pnpm to yarn?

### DISCUSSION THREAD

**Discussion by kerituni12** [toeverything/blocksuite#8387] [General]
In previous versions we used pnpm so is there any problem when using pnpm to switch to yarn?

**Comment by doodlewind** [comment]
Hi, pnpm is great (and even faster in my device)! But since the main downstream project [AFFiNE](https://github.com/toeverything/AFFiNE) is using yarn, switching to yarn makes it easier to debug these two projects together.

For community developers, there shouldn't be noticeable difference.
